### DISCUSSION TITLE
[Unity]puerts有没有类似xlua的机制, 将Vector3的方法实现改为lua实现

### DISCUSSION THREAD

**Discussion by lybugger** [Tencent/puerts#1494] [Q&A]
XLua可以使用xlua.setclass(CS.UnityEngine, "Vector3", LuaVector3), 直接替换lua中受到的Vector3.
比如game Object.transform.position 返回的就是Lua实现的Vector3.
我看现在的puerts还没有类似的功能

**Comment by zombieyang** [comment]
这个感觉业务侧实现也不难...
